### ISSUE TITLE
wrong formatting

### ISSUE THREAD

**Issue by quantrpeter** [antlr/Antlr4Formatter#36]
This
```
r8  	returns[int operandSize,int col]:
```

will become
```
r8 returns [ i n t   o p e r a n d S i z e , i n t   c o l ]
```

Many spaces inserted between characters, which is wrong.
thanks

**Comment by teverett** [comment]
@quantrpeter you are correct.  I don't have time to resolve the issue right now.  A PR would be appreciated.

**Comment by matthiasbalke** [comment]
@quantrpeter We already noticed that. It's the last bug before we can release a first stable release. It's already addressed in: https://github.com/antlr/Antlr4Formatter/issues/25

If you have some spare time to fix this, we would love to accept a PR and finish the release. This is also a prerequisite for merging the spotless integration: https://github.com/diffplug/spotless/pull/328

**Comment by matthiasbalke** [comment]
@quantrpeter This bug should be fixed in version `1.2.1`. Would you mind trying the new version?

**Comment by teverett** [comment]
@quantrpeter could you please verify with version 1.2.1?
